dotnet/SK-dotnet.sln

```diff
@@ -803,6 +807,18 @@ Global
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Publish|Any CPU.Build.0 = Debug|Any CPU
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{FDEB4884-89B9-4656-80A0-57C7464490F7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.ActiveCfg = Debug|Any CPU
+		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Little issue, but it causes big problems in our release pipeline :) It's not important for test projects, but it's important for projects that we pack and release.
```suggestion
		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.ActiveCfg = Publish|Any CPU
		{6744272E-8326-48CE-9A3F-6BE227A5E777}.Publish|Any CPU.Build.0 = Publish|Any CPU
```